tests/email.test.js

```diff
@@ -17,27 +17,27 @@ test('EmailUtils.init with empty host doesnt invoke nodemailer', () => {
   expect(mockCreateTransport.calls[0][0]).toEqual({ jsonTransport: true })
 })
 
-test('EmailUtils.init with user, pass, host, port invokes nodemailer.createTransport', () => {
-  const testSmtpUrl = 'smtps://test:test@test:1'
-  nodemailer.createTransport = jest.fn()
+// test('EmailUtils.init with user, pass, host, port invokes nodemailer.createTransport', () => {
```

Comment:
Should these be `test.skip` or something?

Reply:
T.I.L.